homeassistant/components/zodiac/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "domain": "zodiac",
+  "name": "Zodiac",
+  "documentation": "https://www.home-assistant.io/integrations/zodiac",
+  "codeowners": ["@JulienTant"],
+  "quality_scale": "internal"
```

Comment:
this shouldnt be set to internal, thats reserved for core components. it can be silver, gold, or platinum depending on the rating that your integration meets on the scale. Otherwise, it should be left out of manifest entirely.